contracts/TokenConversionLib.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity 0.4.23;
+
+import "./SafeMath.sol";
+
+library TokenConversionLib {
+    using SafeMath for uint256;
+
+    /**
+     *	@notice Calculate Utility token amount based on stake OST amount, conversion rate and conversion decimal
+     *
+     *	@param amountST amount of ST needs to be staked and converted to UT
+     *	@param conversionRate rate which is used to convert ST to UT
+     *	@param conversionRateDecimals represents number of decimal in conversion rate
+     *
+     *	@return  amountUT number of utility token which can be minted from given amountST
+     */
+
+    function calculateUTAmount(
+        uint256 amountST,
+        uint256 conversionRate,
+        uint8 conversionRateDecimals)
+        internal
+        pure
+        returns(
+        uint256 amountUT)
+    {
+        require(amountST > 0, 'Amount that needs to be converted should be greater than zero');
+        require(conversionRate > 0, 'Conversion Rate should be greater than zero');
+
+        amountUT = (amountST.mul(conversionRate)).div(10 ** uint256(conversionRateDecimals));
```

Comment:
Do you need to set a variable, `amountUT` and return it? Couldn't you simply return `amountST.mul(conversionRate)).div(10 ** uint256(conversionRateDecimals)`?

The question above is less a suggestion of something _to do_ and more a suggestion of something to ask Pro about re: style guide. The question should also include whether we require _naming_ returns (particularly in this kind of library context).